libs/src/services/creatorNode/index.js

```diff
@@ -25,6 +25,23 @@ class CreatorNode {
    */
   static getEndpoints (endpoints) { return endpoints ? endpoints.split(',') : [] }
 
+  /**
+   * Pulls off the user's clock value from a creator node endpoint and the user's wallet address.
+   * @param {string} endpoint creator node endpoint
+   * @param {string} wallet user wallet address
+   */
+  static async getClockValue (endpoint, wallet) {
+    try {
+      return (await axios({
+        url: `/users/clock_status/${wallet}`,
+        method: 'get',
+        baseURL: endpoint
```

Comment:
we should have a timeout here and handle that case

Reply:
Added a flag for timeout.